src/plays/typing-speed-test/components/ResultModal.js

```diff
@@ -0,0 +1,36 @@
+import { Modal } from "@mui/material";
+import "../styles.css";
+
+const ResultModal = ({ open, handleModalClose, wpm, cpm, accuracy }) => {
+  return (
+    <Modal open={open} onClose={handleModalClose}>
+      <div className="result__modal ">
+        <div className="flex flex-col justify-center items-center">
+          <h1 className="text-center text-[1.05rem] tracking-wide my-4 leading-9">
+            {wpm >= 40
+              ? "Excellent..!"
+              : wpm >= 30
+              ? "Good Job..!"
+              : "Nice...!"}{" "}

```

Comment:
Can we handle this logic in util?
We can create on util method which will take the `wpm` as input and return the text.